src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -619,13 +619,13 @@ MStatus PRTModifierAction::updateDynamicEnums() {
 
 		const std::wstring prefix = attrStyle + prtu::STYLE_DELIMITER + attrImport;
 
-		const wchar_t* valuesAttr = (MString(prefix.c_str()) + e.mValuesAttr).asWChar();
-		const prt::Attributable::PrimitiveType type = mGenerateAttrs->getType(valuesAttr);
+		const std::wstring valuesAttr = (MString(prefix.c_str()) + e.mValuesAttr).asWChar();

```

Comment:
Optimization: consider using the prefix above as "string builder" and append the `e.mValuesAttr`, i.e. no need to construct the temporary MString in the first place

Reply:
appending string now